### PR TITLE
Feature/14346 refactoring show error message component

### PR DESCRIPTION
**Pull request by suraj49470**
refactored code to move component level info/error message to page level info/error message

### CODE REVIEW

File: src/app/features/exchange-set/ess-info-error-message/ess-info-error-message.component.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  displayErrorMessage = true;
+  messageType: 'info' | 'warning' | 'success' | 'error' = 'info';
+  messageDesc: string | [];
+  requestedProductsNotInExchangeSet: RequestedProductsNotInExchangeSet[] | undefined;
+  essInfoErrorMessagesubscription: Subscription;
+  constructor(private essInfoErrorMessageService: EssInfoErrorMessageService) {
+  }
+
+  ngOnInit(): void {
+    // eslint-disable-next-line max-len
```
Comment (JiviteshT) [inline src/app/features/exchange-set/ess-info-error-message/ess-info-error-message.component.ts:22]: Remove commented lines

Comment (suraj49470) [reply src/app/features/exchange-set/ess-info-error-message/ess-info-error-message.component.ts]: removed commented lines

File: src/app/core/services/ess-info-error-message.service.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+export interface RequestedProductsNotInExchangeSet{
+  productName: string;
+  reason: string;
+}
+export interface InfoErrorMessage {
+  showInfoErrorMessage: boolean;
+  messageType: 'info' | 'warning' | 'success' | 'error';
+  messageDesc: string;
+  messageTitle?: string;
+  requestedProductsNotInExchangeSet?: RequestedProductsNotInExchangeSet[];
```
Comment (rvijayavel) [inline src/app/core/services/ess-info-error-message.service.ts:13]: As discussed, kindly check whether we need this property or not.

Comment (suraj49470) [reply src/app/core/services/ess-info-error-message.service.ts]: requestedProductsNotInExchangeSet removed

**Review by rvijayavel — APPROVED** [review]
Looks good.

**Review by JiviteshT — APPROVED** [review]
Looks good

**Review by martyn-fewtrell — APPROVED** [review]
Looks good

**Pull request merged**